GitCommands/StringExtensions.cs

```diff
@@ -189,6 +190,18 @@ public static string Quote(this string? s, string q = "\"")
             return string.IsNullOrEmpty(s) ? s : s.Quote();
         }
 
+        /// <summary>
+        /// Quotes and escapes this string for use as a command line argument.
+        /// </summary>
+        [Pure]
+        public static string QuoteForCommandLine(this string s, bool? forWindows = null)
+        {
+            return $"\"{((forWindows ?? EnvUtils.RunningOnWindows()) ? EscapeForWindowsCommandLine(s) : EscapeForPosixCommandLine(s))}\"";
```

Comment:
```suggestion
            return $"\"{(forWindows ?? EnvUtils.RunningOnWindows()) ? EscapeForWindowsCommandLine(s) : EscapeForPosixCommandLine(s)}\"";
```

Reply:
Try it. It's a syntax error. The brackets are mandatory.

Reply:
Sorry, Intellisense did not indicate an error.